### PR TITLE
Support specified columns for full-text index

### PR DESCRIPTION
**Pull request by bright-starry-sky**
1, Changed full-text index structure, deleted field schema_id
2, Support for creating full-text index against fields
3, Support drop full-text index
4, Support show full-text index
5, Refuse to create full-text index if fixed_string type length > 256 
6, Refuse to drop or alter tag if full-text index exists.

depends on 
* vesoft-inc/nebula-common#530
* vesoft-inc/nebula-storage#460

### CODE REVIEW

**Comment by bright-starry-sky** [comment]
ready to review.

File: src/parser/parser.yy [hunk 1]
```diff
@@ -2276,6 +2277,21 @@ create_edge_index_sentence
     }
     ;
 
+create_fulltext_index_sentence
+    : KW_CREATE KW_FULLTEXT KW_INDEX name_label KW_ON KW_TAG name_label L_PAREN name_label_list R_PAREN {
```
Comment (yixinglu) [inline src/parser/parser.yy:2281]:
```
CREATE FULLTEXT TAG INDEX ft_player_name_index ON player(name);
CREATE FULLTEXT EDGE INDEX ft_like_likeness_index ON like(likeness);
```

Comment (bright-starry-sky) [reply src/parser/parser.yy]: > ```
> CREATE FULLTEXT TAG INDEX ft_player_name_index ON player(name);
> CREATE FULLTEXT EDGE INDEX ft_like_likeness_index ON like(likeness);
> ```

Good advice.

**Comment by bright-starry-sky** [comment]
Addressed Yee's comments.

**Review by critical27 — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]

**Pull request merged**